src/app.ts

```diff
@@ -66,12 +73,14 @@ export async function getDefaultAppDependenciesAsync(
         meshClient,
         orderBookService,
         swapService,
+        metaTransactionService,
         provider,
         websocketOpts,
     };
 }
 /**
- * starts the app with dependencies injected
+ * starts the app with dependencies injected. This entry-point is used when running a single instance 0x API
+ * deployment and in tests. It is not used in production deployments where scaling is required.
```

Comment:
Is it used for Kovan FWIW.

Reply:
Hence: "It is not used in production deployments **where scaling is required**" ;)